bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaAdjustableSlatsRollerShutterHandler.java

```diff
@@ -87,7 +87,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             if (OnOffType.ON.equals(command)) {
                 sendCommand(COMMAND_SET_ROCKERPOSITION);
             }
-        } else {
+        } else if (CLOSURE_AND_ORIENTATION.equals(channelUID.getId())) {
+            String param = "[" + command.toString() + "]";
+            sendCommand(COMMAND_SET_CLOSURE_ORIENTATION, param);
+        }else {
```

Comment:
Please apply the formatter (e.g. by running `mvn spotless:apply` in the bindings folder).

Reply:
Applied the formatter